### ISSUE TITLE
NaN rows deleted in categorical option

### ISSUE THREAD

**Issue by Ryo-F** [tompollard/tableone#79]
Hi,
I've been using Tableone til I ran into this problem,

When using `Tableone(categorical=categorical_columns)`,
if categorical_columns contains more than two columns and each of them contains NaN rows,
some rows would be deleted due to dropna() in 
[tableone.py#L468](https://github.com/tompollard/tableone/blob/a7b28ae8099487924cd295dd20def5dfc29c69cf/tableone.py#L468)

I've created a PR to fix this problem, please take a look!

**Comment by tompollard** [comment]
Thanks for picking this up Ryo, we'll take a look!

**Comment by tompollard** [comment]
To reproduce:

```
import random
random.seed(1)
from tableone import TableOne

fruit = ['apple','banana','orange','pineapple','lemon','durian','peach']
n = 4
fruit = [random.sample(fruit, n),
        random.sample(fruit, n),
        random.sample(fruit, n),
        random.sample(fruit, n),
        random.sample(fruit, n),
        random.sample(fruit, n),
        random.sample(fruit, n)]
df = pd.DataFrame(fruit)
df.columns = ['basket1','basket2','basket3','basket4']
df

t1 = TableOne(df, categorical = ['basket1','basket2','basket3','basket4'])
t1

df.loc[1:3,'basket2'] = None
df.loc[2:4,'basket3'] = None

t2 = TableOne(df, categorical = ['basket1','basket2','basket3','basket4'])
t2
```

In  t2, rows with the nulls are missing.

**Comment by tompollard** [comment]
Thanks again @Ryo-F. Fixed in version 0.6.0! I'll keep this issue open until we have added some tests to https://github.com/tompollard/tableone/blob/master/test_tableone.py.
